<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Results</class>
 <widget class="QDialog" name="Dialog_Results">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>261</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="rowCount">
    <number>5</number>
   </property>
   <property name="columnCount">
    <number>1</number>
   </property>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column/>
  </widget>
  <widget class="QLabel" name="label_Res">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>30</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Результат</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Name">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Тестируемый:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Score">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Оценка:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Passed">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>180</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Времени прошло:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Left">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>230</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Времени осталось:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_bestName">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>350</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_bestScore">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>400</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_bestCounter">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>450</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Record">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>280</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Рекорд</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
